agents.ipynb

```diff
@@ -134,7 +141,7 @@
    },
    "source": [
     "# PROGRAM - BlindDog #\n",
-    "Now that we have a <b>Park</b> Class, we need to implement a <b>program</b> module for our dog. A program controls how the dog acts upon it's environment. Our program will be very simple, and is shown in the table below.\n",
+    "Now that we have a <b>Park</b> Class, we need to implement a <b>program</b> module for our dog. A program controls how the dog acts in it's environment. Our program will be very simple, and is shown in the table below.\n",
```

Comment:
Nice catch on the 'in', but it also should be followed by 'its', with no apostrophe.

Reply:
Thanks for the review.
Corrected it along with some other typos.